Strategies/IITB_KPIs.md

```diff
@@ -14,35 +14,77 @@ ESDC services provided to the public or to other departments will need to be del
 
 The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) as reference point and monitor the rate at which they are transformed to digital
 
-`percentage = digital services / total services`
+`service API percentage = digital services / total services`  
 
 ## 2. **Rate of decrease in processing time of ESDC Services**
 
 The processing time of ESDC services that are provided to the public, to other departments, or to other groups within ESDC, will be an indication of improvement. The measurement will be the reduction in time, not the time itself, focusing on continuous improvement rather than a hard target.
 
 The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and monitor their total processing time from beginning to end. This amount, over time, will be monitored to indicate reduction in processing time.
 
-`rate of service improvement = (New Processing Time - Old Processing Time) / Old Processing Time`
+`rate of service improvement = (processing time year_n - processing time year_n-1) / processing time year_n-1`
 
 ## 3. **Reduction in toil of IITB and ESDC Services**
 
 Toil is the kind of work tied to running a production service that tends to be manual, repetitive, automatable, tactical devoid of enduring value, and that scales linearly as a service grows. Processing time of ESDC services may decrease by increasing manual labour but that would defeat the purpose of digital and jeopardize the service's sustainability.
 
 The indicator will be measured two ways:
 
-1. For toil in IITB Services, using ??? and measure the ratio of X over Y.
+1\. Toil in IITB Services
 
-`ratio = X / Y`
+We can measure both the service and development sides:
 
-2. For toil in ESDC Services, using the [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) and measure the ratio of service requests to amount of personnel.
+For service requests (e.g. Helpdesk):
 
-`ratio = service requests / service personnel`
+`service ratio = service requests / service personnel`  
+
+For the development side, there would be multiple metrics, most of them being influenced by the evolution toward DevOps:
+
+**Throughput**
+- `release ratio = code deployments / dev personnel`
+- `lead time for changes ratio =  time from code commit to running in production / dev personnel`
+- `test ratio = tests done / development personnel`
+- `*other?*`
+
+**Stability**
+- `time to restore ratio = time to restore service when incident occurs / dev personnel`
+- `change fail rate ratio = % of changes resulting in degraded service or remediation required / dev personnel`
+- `*other?*`
+
+**Efficiency**
```

Comment:
I worry we go too detailed in measurement. If we can stay strategic and influence other teams to achieve these strategic goals (e.g. ops team, testing team, governance with change approvals) we can let them measure how best to improve.
Recommendations:
1. Take out the manual work ratio and the categories underneath. We keep at the Throughput and Stability level only.
2. Time spent ratio is grouped under a "Quality Outcome" in the DORA report. Create a new KPI about Quality Work that involves the categories mentioned that also aligns with the Phoenix Project's 4 categories of work: business projects, IT projects (internal I&O), changes (bug fixes and security issues), unplanned work

From the DORA Accelerate report, this Efficiency section revolves around Quality.
This particular KPI invovles reducing toil